distribution/lib/Standard/Visualization/0.0.0-dev/src/Warnings.enso

```diff
@@ -12,4 +12,10 @@ process_to_json_text : Any -> Text
 process_to_json_text value =
     warnings = Warning.get_all value
     text = warnings.map w->w.value.to_display_text
-    text.to_json
+    case Warning.reached_max_count value of
+        True ->
+          len = text.length
+          vec = Vector.new (len+1) (i-> if i < len then text.at i else "Warnings limit reached.")
+          vec.to_json
+        False ->
+          text.to_json
```

Comment:
I'd simplify that to this:

```suggestion
    vec = case Warning.reached_max_count value of
        True -> text + ["Warnings limit reached."]
        False -> text
    vec.to_json
```

Also probably rename `text` to `texts` (I know its not coming from you, but I only now noticed it and IMO the singular `text` is a confusing name for a Vector).